### PR TITLE
Order page rework

### PR DESCRIPTION
**Pull request by L3RAZ**

### CODE REVIEW

File: controllers/admin/AdminAjaxPrestashopCheckoutController.php [hunk 1]
```diff
@@ -461,6 +461,8 @@ public function ajaxProcessFetchOrder()
             'moduleName' => $this->module->displayName,
             'orderPayPal' => $presenter->present(),
             'orderPayPalBaseUrl' => $this->context->link->getAdminLink('AdminAjaxPrestashopCheckout'),
+            'moduleLogoUri' => $this->module->getPathUri() . 'logo.png',
+            'orderPaymentLogoUri' => $this->module->getPathUri() . 'views/img/' . strtolower($order->payment) . '.svg',
```
Comment (L3RAZ) [inline controllers/admin/AdminAjaxPrestashopCheckoutController.php:465]: Not sure if this is the right way to select payment icon

File: src/Presenter/Order/OrderPresenter.php [hunk 2]
```diff
@@ -225,4 +234,102 @@ private function getTransactionType($type)
             'class' => $class,
         ];
     }
+
+    private function getTotal()
```
Comment (L3RAZ) [inline src/Presenter/Order/OrderPresenter.php:238]: Would this calculate order total correctly? I couldn't manage to test with few transactions with different currecies.

File: views/templates/admin/ajaxPayPalOrder.tpl [hunk 3]
```diff
@@ -16,161 +16,218 @@
...
+                  <dd>- {$orderPayPalTransaction.paypal_fee} {$orderPayPalTransaction.currency}</dd>
+                  <dt>{l s='Net amount' mod='ps_checkout'}</dt>
+                  <dd>{$orderPayPalTransaction.net_amount} {$orderPayPalTransaction.currency}</dd>
+                </dl>
+              </div>
+              <a href="https://www.paypal.com/activity/payment/{$orderPayPalTransaction.id|escape:'html':'UTF-8'}" target="_blank" class="tabpanel__cta">
+                {l s='See on PayPal' mod='ps_checkout'}
+              </a>
+                {if $orderPayPalTransaction.isRefundable}
+                  <button type="button" class="btn btn-primary btn-sm refund" data-transaction-id="{$orderPayPalTransaction.id|escape:'html':'UTF-8'}">
```
Comment (L3RAZ) [inline views/templates/admin/ajaxPayPalOrder.tpl:115]: Change this button position after mock is updated

File: controllers/admin/AdminAjaxPrestashopCheckoutController.php (hunk 1 above)
Comment (Matt75) [reply controllers/admin/AdminAjaxPrestashopCheckoutController.php:465]: PayPal recommends to use https://github.com/paypal/paypal-sdk-logos
But we needs to use webpack like we do for front.js
It means create a third JS app... 🤯
So I think we can use your solution to stay simple

File: src/Presenter/Order/OrderPresenter.php (hunk 2 above)
Comment (Matt75) [reply src/Presenter/Order/OrderPresenter.php:238]: It should be tested with different currencies, it was designed only for EUR... 😅

**Review by Matt75 — APPROVED** [review]

**Pull request merged**